### PR TITLE
Silence "field mounts[%d].virtiofs.queueSize is only supported on Linux" when it is not specified

### PR DESCRIPTION
**Pull request by AkihiroSuda**

### CODE REVIEW

File: pkg/limayaml/defaults.go [hunk 1]
```diff
@@ -552,7 +567,7 @@ func FillDefault(y, d, o *LimaYAML, filePath string) {
 		if mount.NineP.Msize == nil {
 			mounts[i].NineP.Msize = pointer.String(Default9pMsize)
 		}
-		if mount.Virtiofs.QueueSize == nil {
+		if mount.Virtiofs.QueueSize == nil && *y.VMType == QEMU && *y.MountType == VIRTIOFS {
```
Comment (jandubois) [inline pkg/limayaml/defaults.go:570]: Still needs to check `GOOS`:

```suggestion
		if runtime.GOOS == "linux" && mount.Virtiofs.QueueSize == nil && *y.VMType == QEMU && *y.MountType == VIRTIOFS {
```

Comment (AkihiroSuda) [reply pkg/limayaml/defaults.go:570]: `QEMU && VIRTIOFS` on non-Linux is checked in another place

Comment (jandubois) [reply pkg/limayaml/defaults.go:570]: Yeah, but sometimes too late:

```console
$ l edit alpine --set '.mountType="virtiofs"'
WARN[0000] `--set` is experimental
WARN[0000] field mounts[0].virtiofs.queueSize is only supported on Linux
WARN[0000] field mounts[1].virtiofs.queueSize is only supported on Linux
WARN[0000] field mounts[2].virtiofs.queueSize is only supported on Linux
INFO[0000] Instance "alpine" configuration edited
? Do you want to start the instance now?  (Y/n)
FATA[0001] interrupt
```

But I agree that checking for `GOOS` here is probably not right.

**Review by jandubois — APPROVED** [review]
LGTM

**Pull request merged**